package.json

```diff
@@ -75,6 +75,7 @@
     "datatables.net": "~1.10.12",
     "datatables.net-bs": "~1.10.12",
     "diff": "~3.0.0",
+    "file-saver": "^2.0.1",
```

Comment:
could we get ack from @theforeman/packaging team? (cc @ekohl explicitly as I'm not sure if this is the correct handle :-)

Reply:
I know, but it is messing with the DOM (adding a link and clicking it, sounds like not really clean way to do things).
But if you don't like this one, we already have the download link (for automatic dowload failures), we can use that one. directly and let the browser download the file.